pkg/vulnsrc/mariner/oval/oval.go

```diff
@@ -21,6 +22,11 @@ func ParseDefinitions(dir string) ([]Definition, error) {
 	err := utils.FileWalk(dir, func(r io.Reader, path string) error {
 		var def Definition
 		if err := json.NewDecoder(r).Decode(&def); err != nil {
+			// Definition.Metadata.Patchable has a bool and "Not Applicable".
+			// Workaround: Ignore "Not Applicable"
+			if strings.HasSuffix(err.Error(), "unmarshal \"Not Applicable\" into bool") {
```

Comment:
https://github.com/golang/go/blob/go1.18/src/encoding/json/decode.go#L919
Also this error.

I use HasSuffix, is there a better way?